### ISSUE TITLE
xarray interactive `sel` accessor can't handle more than one widget

### ISSUE THREAD

**Issue by nitrocalcite** [holoviz/hvplot#593]
Trying to use hvplot's interactive xarray `sel` fails if you have more than one widget.
I can't find examples of this usage i.e. [here](https://hvplot.holoviz.org/user_guide/Interactive.html), so maybe it's not intended to be supported, but it surprised me to see it break.

Here's a MRE in Jupyter notebook
```
import xarray as xr
import hvplot.xarray
import panel.widgets as pnw

ds = xr.tutorial.load_dataset('air_temperature')
ds.interactive.sel(lon=pnw.DiscreteSlider, lat=pnw.DiscreteSlider)
```
Dragging the `lat` slider works, but the `lon` slider has no effect; I expected both of them to change the dataset.

On further testing, it seems if specifying n > 1 sliders, only the last slider will have an effect.
This seems to be an interface limitation - I assume it's all built from `@pn.depends` or similar - but also, chaining `.sel` works just fine:
```
ds.interactive.sel(lon=pnw.DiscreteSlider).sel(lat=pnw.DiscreteSlider)
```

Let me know if there's any other information I can provide!  Thanks for the awesome interface!

#### ALL software version info
`hvplot 0.7.1`
`holoviews 1.14.2`
`xarray 0.16.2`
Chrome on Windows

**Comment by jbednar** [comment]
Multiple widgets are definitely meant to be supported, so this seems to be a bug and maybe a regression.

**Comment by nitrocalcite** [comment]
Just reproduced the same behavior on `hvplot 0.7.0`, though in theory it could be a regression related to a different package version.

**Comment by philippjfr** [comment]
Definitely looks like a bug to me.

**Comment by nitrocalcite** [comment]
Yup, looks like a typo in holoviews.  The key observation is that `ids0._transform.ops` contains both widgets in one `op`, whereas `ids1._transform.ops` has it in two.  Due to the holoviews bug, `ids0._transform.params` is missing one `param.Parameter`, which causes the `pn.depends` call in `ids0._callback` to be missing it as well (via `ids0._params`).  Without being included in the `depends`, the widget doesn't link properly.

**Comment by nitrocalcite** [comment]
Closing as the patch appears to have been released with holoviews 1.14.3.

**Comment by jbednar** [comment]
Thanks!
